streaming/base/compression/compression.py

```diff
@@ -162,6 +162,7 @@ def decompress(self, data: bytes) -> bytes:
     'bz2': Bzip2,
     'gz': Gzip,
     'snappy': Snappy,
+    'zst': Zstandard,
     'zstd': Zstandard,
```

Comment:
Looks like a duplicate ? Do we still need `zstd` ?

Reply:
"zstd" for backwards compatibility

Reply:
Can you update all the convert scripts and all the test suite from `zstd` to `zst` so that future user don't use `zstd` anymore and in future we can remove `zstd` from out code base!!!

Reply:
reverted usage of zst